README.md

```diff
@@ -7,16 +7,18 @@ This repository contains the alpha-stage data publishing component of data.gov.u
 
 ## Development
 
-To use this repository for development you should run the following tasks:
+To use this repository for development you should run the following tasks to setup the environment:
 
 ```bash
 git clone <REPO>
 cd publish_data_alpha
 vagrant up
 vagrant ssh
 cd /vagrant/tools
-sudo bash dev_run.sh
+sudo bash dev_setup.sh
+. dev_run.sh
```

Comment:
the names are misleading but in fact dev_setup runs the app. Probably should remove the last 3 lines of setup?